src/collider/collider/logmanager.go

```diff
@@ -0,0 +1,143 @@
+// Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file in the root of the source
+// tree.
+
+// Package collider implements a signaling server based on WebSocket.
+package collider
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"strings"
+	"time"
+)
+
+// LogManager redirects log output to files, one for each |cycle|, and deletes files that are last modified before the retention period.
```

Comment:
With the disclaimer that I don't know Go style - this comment seems to exceed 80 chars.